backends/events/sdl/resvm-sdl-events.cpp

```diff
@@ -115,7 +130,7 @@ bool ResVmSdlEventSource::handleKbdMouse(Common::Event &event) {
 			_km.x = oldKmX;
 			_km.y = oldKmY;
 		}
-
+		else
```

Comment:
What does this change fix?

Reply:
Without it, the camera becomes crazy when moving the joystick in Myst3 (on RPI) and it directly points to the zenith !
I think it's linked to a bug of warpMouse in SDL2 (2.0.8 and lower versions).

I'll recheck and put a descriptive comment.

Reply:
I can confirm this change fixes the same behavior in my WIP Nintendo Switch port.

Reply:
Ok, it makes sense to me now. I suspect SDL generates a mouse moved event when `warpMouseInWindow` is called.

Can you please write the code as:
```cpp
if (graphicsManager) {
    if (graphicsManager->isMouseLocked()) {
       _km.x = oldKmX;{
       _km.y = oldKmY;
    } else {
        graphicsManager->getWindow()->warpMouseInWindow((Uint16)(_km.x / MULTIPLIER), (Uint16)(_km.y / MULTIPLIER));
    }
}
```